objects/src/accounts/component/template/storage/mod.rs

```diff
@@ -0,0 +1,585 @@
+use alloc::{boxed::Box, string::String, vec::Vec};
+
+use vm_core::{
+    utils::{ByteReader, ByteWriter, Deserializable, Serializable},
+    Word,
+};
+use vm_processor::{DeserializationError, Digest};
+
+mod word;
+pub use word::*;
+
+use super::AccountComponentTemplateError;
+use crate::accounts::{StorageMap, StorageSlot};
+
+mod placeholder;
+pub use placeholder::{StoragePlaceholder, StorageValue};
+
+mod init_storage_data;
+pub use init_storage_data::InitStorageData;
+
+#[cfg(feature = "std")]
+pub mod toml;
+
+// STORAGE ENTRY
+// ================================================================================================
+
+/// Represents a single entry in the component’s storage layout.
+///
+/// Each entry can describe:
+/// - A value slot (single word or multiple words).
+/// - A map slot (key-value map that occupies one storage slot).
+/// - A multi-slot entry (spanning multiple contiguous slots, with multiple values).
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub enum StorageEntry {
```

Comment:
I think doc comments changes here may not have been committed. i.e., we still mention "multiple words" on line 30, we still should probably mention that multi-slot entries are only for values (not maps).

Reply:
Yes, I think I rolled back some changes too far back and did not re-add them